rasterio/_crs.pxd

```diff
@@ -0,0 +1,8 @@
+# _CRS class definition
+
+include "gdal.pxi"
+
+
+cdef class _CRS:
+
+    cdef OGRSpatialReferenceH _osr
```

Comment:
Here's the essence of it! Now we get C allocation and deallocation at the cost of not being able to derive from an ordinary Python class.